src/configureWorkspace/configureDotNetCore.ts

```diff
@@ -302,10 +302,9 @@ function generateComposeFiles(dockerfileName: string, platform: Platform, os: Pl
         }
     }
 
-    let volumesList = '      - ~/.vsdbg:/remote_debugger';
-    if (os === 'Linux') {
-        volumesList += ':rw'
-    }
+    let volumesList = os === 'Windows' ?
+        '      - ~/.vsdbg:c:\\remote_debugger'
```

Comment:
```suggestion
        '      - ~/.vsdbg:c:\\remote_debugger:rw'
```

The `:rw` works on Windows without issue. We should keep it for consistency.

Reply:
(By the way, I just learned about this suggest changes feature and it's really neat! Makes it easy to incorporate changes from feedback)